### PR TITLE
dnsmonitor 1.0.1 (new cask)

### PR DESCRIPTION
**Pull request by kohnakagawa**
A utility for monitoring DNS requests and responses on macOS

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/d/dnsmonitor.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "dnsmonitor" do
+  version "1.0.1"
+  sha256 "ba0a54877cea0e3731271621ee09f4a6877548b5bdcbf785ba38862600bbabaf"
+
+  url "https://github.com/objective-see/DNSMonitor/releases/download/v#{version}/DNSMonitor_#{version}.zip",
+      verified: "github.com/objective-see/DNSMonitor/"
+  name "dnsmonitor"
+  desc "Monitor DNS activity"
+  homepage ""
```
Comment (SMillerDev) [inline Casks/d/dnsmonitor.rb:9]:
```suggestion
  homepage "https://github.com/objective-see/DNSMonitor"
```

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by krehel** [comment]
n.b the reason I removed the `binary` line is because of this note from the authors site.  Using the binary also didn't seem to work for the reason below.

> 
> Though DNSMonitor is distributed as an application (in order to satisfy Apple's codesigning, notarization, and System/Network extension requirements), it is a CLI utility. As such, it should be executed from the commandline, by specifying the path to its application binary:
> 
> DNSMonitor.app/Contents/MacOS/DNSMonitor
> 
> Moreover, it must remain in its application bundle (which contains required codesigining and provisioning profile information).

I could not find any files to `zap` so marked as not required.

File: Casks/d/dnsmonitor.rb (hunk 1 above)
Comment (krehel) [reply Casks/d/dnsmonitor.rb]: Added this as part of my update

**Review by p-linnane — APPROVED** [review]

**Pull request merged**